waku/node/peer_manager/peer_manager.nim

```diff
@@ -49,10 +49,10 @@ const
   BackoffFactor = 4
 
   # Limit the amount of paralel dials
-  MaxParalelDials = 10
+  MaxParallelDials = 10
 
   # Delay between consecutive relayConnectivityLoop runs
-  ConnectivityLoopInterval = chronos.seconds(15)
+  ConnectivityLoopInterval = chronos.seconds(60)
```

Comment:
note that this will increase the time it takes to get a reasonable amount of peers at startup. it will only attempt 10 peers each 1 minute, so if we set eg 50 peers, it will take 5 minutes, which seems quite a lot.

Reply:
can see that `connectToRelayPeers` has been modified, so nothing to address then :)